src/babel.js

```diff
@@ -153,24 +153,7 @@ function evaluate(babel, path, state) {
       path,
       babel.types,
       state.file.opts.filename,
-      text => {
```

Comment:
@satya164 I don't even remember what was this for, maybe I used it to set `babelrc: false`? 
It was failing because it couldn't find the plugins :)

Reply:
oh because it is using your preset I guess :)